fastchat/serve/model_worker.py

```diff
@@ -98,6 +112,10 @@ def register_to_controller(self):
             "check_heart_beat": True,
             "worker_status": self.get_status(),
         }
+        if enable_attest:
```

Comment:
Here we provide our quote for verification, I think we need to handle these `quotes`  in `controller.py` accordingly?

Reply:
Delete this